### PR TITLE
Document bzz-list, bzz-raw and bzz-immutable schemes

### PR DESCRIPTION
**Pull request by janos**
PR https://github.com/ethereum/go-ethereum/pull/15667 has been merged to ethereum master and this change updates the documentation.

### CODE REVIEW

File: contents/usage.rst [hunk 1]
```diff
@@ -204,6 +204,19 @@ The possibility to POST makes swarm an actual cloud service, bringing upload fun
...
 In fact the command line tool ``swarm up`` uses the http proxy with the bzz raw scheme under the hood.
 
+bzz-list
+^^^^^^^^^^^^^^
+
+.. code-block:: none
+
+    GET http://localhost:8500/bzz-list:/2477cc8584cc61091b5cc084cdcdb45bf3c6210c263b0143f030cf7d750e894d
+
+Returns a list of all files contained in <manifest> under <path> grouped into common prefixes using ``/`` as a delimiter. The response is a JSON-encoded object with ``common_prefixes`` string field and ``entries`` list field.
```
Comment (lmars) [inline contents/usage.rst]: This comment refers to `<manifest>` and `<path>` but they don't appear in the example?

File: contents/usage.rst [hunk 2]
```diff
@@ -204,6 +204,19 @@ The possibility to POST makes swarm an actual cloud service, bringing upload fun
...
+
+bzzr and bzzi
+^^^^^^^^^^^^^^
+Schemes with short names bzzr and bzzi are deprecated in favour of bzz-raw and bzz-immutable, respectably. They are kept for backward compatibility, and will be removed on the next release.
```
Comment (lmars) [inline contents/usage.rst]: s/respectably/respectively

Comment (janos) [reply contents/usage.rst]: :( Thank you.

File: contents/usage.rst (hunk 1 above)
Comment (janos) [reply contents/usage.rst]: Yes, the description is not so clear with this example. Updated.

File: contents/simpleuser.rst [hunk 3]
```diff
@@ -157,11 +157,11 @@ Files can also be uploaded using a simple HTML form:
 Listing files
 -------------
 
-Setting `list=true` in the query of a `GET` request returns a list of files contained under the path, grouped into common prefixes which represent directories:
+A `GET` request with bzz-list url scheme returns a list of files contained under the path, grouped into common prefixes which represent directories:
```
Comment (gbalint) [inline contents/simpleuser.rst]: Maybe bzz-list can be teletyped or bold for readability

Comment (janos) [reply contents/simpleuser.rst]: Agreed and updated 👍

File: contents/usage.rst [hunk 4]
```diff
@@ -170,32 +170,32 @@ This generic scheme supports name resolution for domains registered on the Ether
...
+bzz-immutable
 ^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: none
 
-    GET http://localhost:8500/bzzi:/2477cc8584cc61091b5cc084cdcdb45bf3c6210c263b0143f030cf7d750e894d
+    GET http://localhost:8500/bzz-immutable:/2477cc8584cc61091b5cc084cdcdb45bf3c6210c263b0143f030cf7d750e894d
 
-The same as the generic scheme but there is no ENS domain resolution, the domain part of the path needs to be a valid hash. This is also a read-only scheme but explicit in its integrity protection. A particular bzzi url will always necessarily address the exact same fixed immutable content.
+The same as the generic scheme but there is no ENS domain resolution, the domain part of the path needs to be a valid hash. This is also a read-only scheme but explicit in its integrity protection. A particular bzz-immutable url will always necessarily address the exact same fixed immutable content.
```
Comment (homotopycolimit) [inline contents/usage.rst]: "also read-only but explicit in its integrity protection" -> this is a strange formulation.

**Pull request merged**